lib/bolt/secret/base.rb

```diff
@@ -23,18 +23,18 @@ def decode(code)
       end
 
       def secret_encrypt(opts)
-        encrypted = encrypt_value(opts['plaintext-value'])
+        encrypted = encrypt_value(opts['plaintext_value'])
         encode(encrypted)
       end
 
       def secret_decrypt(opts)
-        raw, _plugin = decode(opts['encrypted-value'])
+        raw, _plugin = decode(opts['encrypted_value'])
         decrypt_value(raw)
       end
-      alias inventory_config secret_decrypt
+      alias resolve_reference secret_decrypt
 
-      def validate_inventory_config(opts)
-        decode(opts['encrypted-value'])
+      def validate_resolve_reference(opts)
+        decode(opts['encrypted_value'])
```

Comment:
```suggestion
        # TODO: Remove deprecation warning
        raise Bolt::ValidationError, "The 'encrypted-value' key is deprecated migrate to to 'encrypted_value'" if opts.include?('encrypted-value')
        decode(opts['encrypted_value']
```
Can we be helpful here to people who have `encrypted-value`? Otherwise they just see:
```
Could not parse as an encrypted value: 
```